crates/autopilot/src/limit_orders/quoter.rs

```diff
@@ -118,7 +118,14 @@ impl LimitOrderQuoter {
                     full_fee_amount: quote.full_fee_amount,
                 },
             )
-            .await
+            .await?;
+        // In order to compute the rewards for this order while building the auction we need to have
+        // a valid quote for it in our database at that time. That means we have to:
+        // 1) make the quote valid for the entire time where the order could possibly make it into an auction
+        // 2) store the quote in the DB
+        quote.data.expiration = Utc::now() + self.limit_order_age;
+        self.quoter.store_quote(quote).await?;
```

Comment:
nit: Shouldn't we change `update_limit_order_fees` to store both the quote and the fees in an atomic transaction?

Reply:
Specifically, I would change it to just store it in the `order_quotes` table and not in the `quotes` table.

Reply:
Yeah, making that atomic definitely makes sense. 👍 
Will double check the table where we store the quote.